cmd/mavenBuild.go

```diff
@@ -110,6 +110,15 @@ func runMavenBuild(config *mavenBuildOptions, telemetryData *telemetry.CustomDat
 		log.Entry().Warnf("failed to create build settings info: %v", err)
 	}
 	commonPipelineEnvironment.custom.buildSettingsInfo = buildSettingsInfo
+	commonPipelineEnvironment.artifacts, err = maven.FindArtifacts(&maven.EvaluateOptions{
```

Comment:
could create a "last one wins" situation...

Reply:
I think @kulhadia and colleagues did consider that already, but wanted to defer a discussion about changing the CPE to allow adding artifacts (or in fact other things using  array or map like structures).